app/services/oauth_clients/connection_manager.rb

```diff
@@ -31,7 +31,8 @@
 
 module OAuthClients
   class ConnectionManager
-    AUTHORIZATION_CHECK_PATH = '/ocs/v2.php/cloud/capabilities'.freeze
+    # Nextcloud API endpoint to check if Bearer token is valid
+    AUTHORIZATION_CHECK_PATH = '/ocs/v1.php/cloud/user'.freeze
```

Comment:
is this the endpoint gotten from Julien? any idea, why it is a `v1` endpoint?

Reply:
Rrright, from Julien to Wieland to myself.
This is supposably the cheapest way to test for a valid Bearer token.